sbt/common.sbt

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) from 2022 The Play Framework Contributors <https://github.com/playframework>, 2011-2021 Lightbend Inc. <https://www.lightbend.com>
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// NOTE: !!! THIS IS A COPY !!!                                                                                //
+// To edit this file use the main version in https://github.com/playframework/.github/blob/main/sbt/common.sbt //
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * If you need an extra commands to format source code, add the next code to your `build.sbt`
+ * {{{
+ *   val _ = sys.props += ("sbt_formatCode" -> List(...).mkString(";"))
+ * }}}
+ */
```

Comment:
@mkurz If you agree with this comment and below, I will add a synchronization with main Play repo and merge it.

Reply:
Sure! Maybe it's also possible to sync the files into the Play repo's ./documentation/` folder?

Reply:
Yes, it's possible, I'll do that 😉

Reply:
Done ✔️

Reply:
https://github.com/playframework/playframework/pull/11631